web/src/components/expandable-information/expandable-information.tsx

```diff
@@ -0,0 +1,28 @@
+import * as React from 'react';
+import { Localized } from '@fluent/react';
+
+import { ChevronDown } from '../ui/icons';
+
+import './expandable-information.css';
+
+interface Props {
+  summaryLocalizedId: string;
+  children: React.ReactNode;
+}
+
+const ExpandableInformation = ({ summaryLocalizedId, children }: Props) => {
+  return (
+    <details className="expandable-information">
```

Comment:
If we use the `<details>` component we don't need to use React to handle any state management, it works out of the box for accessibility too